lib/pages/settings_dashboard/settings_profile/settings_profile.dart

```diff
@@ -0,0 +1,264 @@
+import 'package:adaptive_dialog/adaptive_dialog.dart';
+import 'package:file_picker/file_picker.dart';
+import 'package:fluffychat/pages/settings_dashboard/settings_dashboard_manager.dart';
+import 'package:fluffychat/pages/settings_dashboard/settings_profile/settings_profile_item_style.dart';
+import 'package:fluffychat/pages/settings_dashboard/settings_profile/settings_profile_view.dart';
+import 'package:fluffychat/presentation/enum/settings/settings_profile_enum.dart';
+import 'package:fluffychat/utils/extension/value_notifier_extension.dart';
+import 'package:fluffychat/utils/platform_infos.dart';
+import 'package:fluffychat/widgets/matrix.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:future_loading_dialog/future_loading_dialog.dart';
+import 'package:image_picker/image_picker.dart';
+import 'package:matrix/matrix.dart';
+import 'package:flutter_gen/gen_l10n/l10n.dart';
+
+class SettingsProfile extends StatefulWidget {
+  final Profile? profile;
+
+  const SettingsProfile({
+    super.key,
+    required this.profile,
+  });
+
+  @override
+  State<SettingsProfile> createState() => SettingsProfileController();
+}
+
+class SettingsProfileController extends State<SettingsProfile> {
+  final settingsDashboardManagerController =
+      SettingsDashboardManagerController();
+
+  final ValueNotifier<bool> isEditedProfileNotifier = ValueNotifier(false);
+
+  Client get client => Matrix.of(context).client;
+
+  String get mxid => settingsDashboardManagerController.mxid(context);
+
+  String get displayName =>
+      settingsDashboardManagerController.displayName(context);
```

Comment:
i think you should use variable for this, use get method, make multiple call to the function again and again ?